app/frontend/js/TestDestination.js

```diff
@@ -0,0 +1,63 @@
+
+import { request } from "./utils/request";
+
+document.addEventListener(
+  "DOMContentLoaded",
+  function () {
+    function alertClass(statusCode) {
+      if (statusCode >= 200 && statusCode < 300) {
+        return "my-2 alert alert-success";
+      } else {
```

Comment:
This could be a ternary operation? 

sorry, couldn't resist 😆 Code looks great  👍

Reply:
Nice try haha